config.sh

```diff
@@ -0,0 +1,70 @@
+function pre_build {
+    echo "Starting pre-build"
+
+    # Any stuff that you need to do before you start building the wheels
+    # Runs in the root directory of this repository.
+
+    # https://github.com/matthew-brett/multibuild/pull/146
+    yum install -y rsync
+
+    echo 'Building fftw'
+    # Taken from: https://github.com/conda-forge/fftw-feedstock/blob/master/recipe/build.sh
+    export CFLAGS="-O3 -fomit-frame-pointer -fstrict-aliasing -ffast-math"
+
+    # TODO: make this a loop:
+    # single
+    echo 'Building fftw: single'
+    build_simple fftw 3.3.7 \
+        http://www.fftw.org/ tar.gz \
+        --with-pic --enable-shared --enable-threads --disable-fortran \
+        --enable-float --enable-sse --enable-sse2 --enable-avx
+    # eval cd tests && make check-local && cd -
+
+    # Clear stamp file which prevents subsequent builds
+    rm fftw-stamp
+
+    # double
+    echo 'Building fftw: double'
+    build_simple fftw 3.3.7 \
+        http://www.fftw.org/ tar.gz \
+        --with-pic --enable-shared --enable-threads --disable-fortran \
+        --enable-sse2 --enable-avx
+    # eval cd tests && make check-local && cd -
+
+    # Clear stamp file which prevents subsequent builds
+    rm fftw-stamp
+
+    # long double (SSE2 and AVX not supported)
+    echo 'Building fftw: long double'
+    build_simple fftw 3.3.7 \
+        http://www.fftw.org/ tar.gz \
+        --with-pic --enable-shared --enable-threads --disable-fortran \
+        --enable-long-double
+    # eval cd tests && make check-local && cd -
+
+    # Taken from: https://github.com/conda-forge/pyfftw-feedstock/blob/master/recipe/build.sh
+    # Should this be in env_vars.sh ?
```

Comment:
It needs to be in `env_vars.sh` only if a) the steps before this need the variable or b) you need it for the test phase as well.  Otherwise, this will work fine to set it in the build environment.